Gruntfile.js

```diff
@@ -427,4 +427,46 @@ module.exports = function(grunt) {
 
 		return {};
 	}
+
+	function loadWebStoreConfig() {
+		if (isTravisCi) {
```

Comment:
This is so we can use the "old way" but if we are on travis do it travis style ;)